src/lib/y2storage/bcache.rb

```diff
@@ -59,6 +64,21 @@ def inspect
       "<Bcache #{name} #{bcache_cset.inspect} -> #{blk_device}>"
     end
 
+    def self.usable_blk_devices(device_graph)
```

Comment:
I would call this partitioner-specific logic. It's the kind of things we have the actions for.

Reply:
ok

Reply:
so action should pass it to dialog and dialog should provide it to widget, right?

Reply:
Yes, that's my idea. Pretty much like it's done here https://github.com/yast/yast-storage-ng/blob/master/src/lib/y2partitioner/actions/create_partition_table.rb#L167